src/validators/form-validator.ts

```diff
@@ -24,33 +26,57 @@ export const FieldValidator: FormFieldValidator = {
       const min = Number(field.questionOptions.min);
       const max = Number(field.questionOptions.max);
       if (isEmpty(value)) return [];
-      return !Number.isNaN(min) || !Number.isNaN(max) ? numberInputRangeValidator(min, max, value) : [];
+      return !Number.isNaN(min) || !Number.isNaN(max) ? numberInputRangeValidator(min, max, value, field) : [];
     }
     return [];
   },
 };
 
-export function numberInputRangeValidator(min: number, max: number, inputValue: number) {
+export function numberInputRangeValidator(min: number, max: number, inputValue: number, field?: FormField) {
   if (!Number.isNaN(min) && inputValue < min) {
     return [
       {
-        resultType: 'error',
+        // t('error', 'error')
+        resultType: translateFrom(moduleName, 'error', 'error'),
         errCode: fieldOutOfBoundErrCode,
-        message: `Value must be greater than ${min}`,
+        // t('maxValue', 'Value must be lower than {{max}}', {max})
+        message: translateFrom(moduleName, 'maxValue', 'Value must be greater than {{max}}', {
+          min,
+        }),
       },
     ];
   }
 
   if (!Number.isNaN(max) && inputValue > max) {
     return [
       {
-        resultType: 'error',
+        // t('error', 'error')
+        resultType: translateFrom(moduleName, 'error', 'error'),
         errCode: fieldOutOfBoundErrCode,
-        message: `Value must be lower than ${max}`,
+        // t('minValue', 'Value must be lower than ${min}', {min})
+        message: translateFrom(moduleName, 'minValue', 'Value must be lower than ${min}', {
+          max,
+        }),
       },
     ];
   }
 
+  if (field.questionOptions.disallowDecimals) {
```

Comment:
What happens if a user forgets to update this value in the form? In such scenarios, we should infer this value from the concept.

Reply:
noted